Public/Src/Utilities/Instrumentation/Common/AriaV2StaticState.cs

```diff
@@ -19,7 +19,7 @@ public static class AriaV2StaticState
         public const int AriaMaxPropertyLength = 100;
 
         private static readonly object s_syncRoot = new object();
-        private static readonly TimeSpan s_defaultShutdownTimeout = TimeSpan.FromSeconds(20);
+        private static readonly TimeSpan s_defaultShutdownTimeout = TimeSpan.FromMinutes(2);

```

Comment:
Can you also update [`AriaLogger.cpp`](https://github.com/microsoft/BuildXL/blob/eebb761925cb7fa986c0ab02f6c52f273ff76090/Public/Src/Utilities/Instrumentation/AriaNative/lib/AriaLogger.cpp#L18) by changing
```c
config[CFG_INT_MAX_TEARDOWN_TIME] = 15;
```
to
```c
config[CFG_INT_MAX_TEARDOWN_TIME] = 2 * 60; // 2 minutes
```

Ideally, the same `s_defaultShutdownTimeout` value should be plumbed all the way through, but that requires recompiling and republishing some binaries for macOS, so I can do that in a separate PR.